chia/full_node/fee_history.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import List
+
+from chia.util.ints import uint8, uint32
+from chia.util.streamable import Streamable, streamable
+
+
+@streamable
+@dataclass(frozen=True)
+class FeeStatBackup(Streamable):
+    type: str
+    tx_ct_avg: List[str]
```

Comment:
can you comment on why we are using strings here ?

i couldn't find where this is saved to disk